tests/integration_tests.rs

```diff
@@ -193,6 +193,40 @@ async fn test_wasi_provider() -> Result<(), Box<dyn std::error::Error>> {
         }
     );
 
+    let secrets: Api<Secret> = Api::namespaced(client.clone(), "default");
+    secrets
+        .create(
+            &PostParams::default(),
+            &serde_json::from_value(json!({
```

Comment:
nit: we should probably move these json values somewhere else so that the test function isn't so verbose. It's become a bit hard to read.

Reply:
What are your thoughts on maybe just defining them as normal pod YAML manifests and then reading them from a file?